pyked/tests/testfile_rcm2.yaml

```diff
@@ -32,12 +32,21 @@ datapoints:
         uncertainty: 0.01
     pressure:
       - 958.0 torr
-    compressed-pressure:
-      - 7.1 bar
-    compressed-temperature:
-      - 765 K
-      - uncertainty: 0.01
-        uncertainty-type: relative
+    rcm-data:
+      compressed-pressure:
+        - 7.1 bar
+      compressed-temperature:
+        - 765 K
+        - uncertainty: 0.01
+          uncertainty-type: relative
+      compression-time:
+        - 38.0 ms
+      stroke:
+        - 10.0 inch
+      clearance:
+        - 2.5 cm
+      compression-ratio:
+        - '12.0'
```

Comment:
are the quote marks needed here?

Reply:
Ah, no, shouldn't be, I was testing a weird failure mode. Thanks for the catch